README.md

```diff
@@ -96,6 +96,17 @@ Execute the following command from the root of the repository to run the tests:
 dart test
 ```
 
+## Applications using this SDK
+
+Here is a list of known apps using this SDK:
+
+- **Smoothie**. [GitHub](https://github.com/openfoodfacts/smooth-app) / Will soon be released
```

Comment:
It would probably good to somehow display that it's from OFF itself also it's already publicly available but afaik it's a older version

Reply:
I have changed the content to differentiate the official app from third parties.
I prefer not to link an Android or iOS release, as we will change the package name (re-use current apps)